lib/landing_session.js

```diff
@@ -187,6 +187,9 @@ class LandingSession extends Session {
     const takeMessage = await cli.prompt('Use this message?');
     if (takeMessage) {
       await runAsync('git', ['commit', '--amend', '-F', messageFile]);
+      cli.log('Please run the following command to verify' +
```

Comment:
This would show up after every `git node land --amend`, is that intended?

Reply:
Also, why this is only printed when `takeMessage` (use the automatically generated message) is true?

Reply:
About message appearing after every `git node land --amend`
* Yes, it's advisable to do this so that new collaborators or ones using node-core-utils infrequently remember to run `git node land --final` before pushing commits.

About why this is only printed when `takeMessage` is true:
* I noticed that when `takeMessage` is false, collaborator is recommended to run `git commit --amend -F ${messageFile}` - so the message to run `git node land --final` will be shown when that command is run.

Reply:
@trivikr 

> I noticed that when takeMessage is false, collaborator is recommended to run git commit --amend -F ${messageFile} - so the message to run git node land --final will be shown when that command is run.

I don't think any code from ncu is going to run after the user runs `git commit --amend -F ...`? Because we don't have any git hook for ordinary git commits.

Reply:
How about recommending user to run following command instead?
`git node land --amend -F ${messageFile}`

which internally calls `git commit --amend -F ${messageFile}`

Reply:
Maybe we can just suggest them to call

```
git commit --amend -F ${messageFile}
git node land --continue
```

?